components/brave_vpn/brave_vpn_service.cc

```diff
@@ -141,7 +142,7 @@ BraveVpnService::BraveVpnService(
     ReloadPurchasedState();
   }
   base::PowerMonitor::AddPowerSuspendObserver(this);
-#endif
+#endif  // !BUILDFLAG(IS_ANDROID)
 }
 
 BraveVpnService::~BraveVpnService() {
```

Comment:
all shutdown should be done in `BraveVpnService::Shutdown`, this includes mojo and weakptr invalidation.

Reply:
Fixed